app/src/main/java/com/example/myapplication/ui/ProductAttributeComposable.kt

```diff
@@ -0,0 +1,169 @@
+package com.example.myapplication.ui
+
+import android.annotation.SuppressLint
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.pager.HorizontalPager
+import androidx.compose.foundation.pager.rememberPagerState
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.Icon
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.material.TopAppBar
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Close
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import com.example.myapplication.R
+import com.example.myapplication.model.ButtonAction
+import com.example.myapplication.model.ProductAttributeData
+import com.example.myapplication.ui.theme.PriceBackground
+import com.example.myapplication.ui.theme.PrimaryColor
+import com.example.myapplication.ui.theme.Shapes
+import com.example.myapplication.ui.theme.Typography
+import com.kole.myapplication.cms.nnsettings.NNSettingsString
+
+@SuppressLint("UnusedMaterialScaffoldPaddingParameter")
+@Composable
+fun ProductAttributeComposable(
+    modifier: Modifier = Modifier,
+    productAttributeData: ProductAttributeData,
+    onButtonClicked: (ButtonAction) -> Unit
+) {
+    Scaffold(
```

Comment:
Can you set up the status bar color to have the same as the screen's background?